.github/workflows/cicd.yaml

```
@@ -90,21 +92,21 @@ jobs:
 
       - name: build sdist and wheel ( no test )
         run: |
-          python -m build --sdist --wheel --outdir .tox/dist/
+          python -m build --sdist --wheel
```

even though it's the `default`, perhaps better explicit ?

```suggestion
          python -m build --sdist --wheel --outdir dist
```